docs/explanation/chisel.rst

```diff
@@ -44,3 +44,14 @@ could see as a block of stone from which we can carve and extract only the
 small and relevant parts that we need to run our applications, thus keeping
 ROCKs small and less exposed to vulnerabilities.
 
+Slice definitions
+~~~~~~~~~~~~~~~~~
+
+A package's slices can be defined via a YAML slice definitions file. Check the
```

Comment:
I feel this need to be done before talking about slice because this section is called ``slice definitions. ``

Reply:
most people will be users of slices and thus never actually define slices. the previous section introduces the concept, while this one drills down into how to define slices. 

"Package slices (what are these?) -> Slice definitions (learn how to create a slice)" seems like a logical flow to me and thus why the latter became a subsection of the former

Reply:
It's maybe the name of the section which confuses me. I think it's better to have @dboddie feedback on this.